### ISSUE TITLE
S3 Dependency can break UI and is potential security risk

### ISSUE THREAD

**Issue by ClaraLeigh** [thedevdojo/voyager#2562]
### Description:
In your app.js you have the call: require('./load-remote.js');
This simply loads a S3 file "https://s3.amazonaws.com/laravelvoyager/voyager.js" with a basic 1 line console.log().

For the page to finish loading, this file has to finish loading first, which causes issues when S3 is offline or you're using a local dev environment that has a slow internet connection (or no internet connection).

In addition to it slowing everything down unnecessarily it is also a security risk.
If someone was to gain access to this S3 bucket they would be able to modify this code and inject anything they want onto the users website.

Can we please just remove this from the code?

**Comment by fletch3555** [comment]
It is there as a means for us to push a message saying "new security update released, please update Voyager" or something like that.  I can't say I 100% agree with the approach, but @tnylea added it (I think?).  I'm thinking we can remove it.

**Comment by ClaraLeigh** [comment]
Okay cool, I found something else was interfering with page load so I'm not sure if that's related as I thought it would be loading async.

Either way, I'm not that comfortable having an external script loaded like this that can be changed by someone else. I've manually modified the minified file to just load a local empty file instead; as when I tried to re-compile everything I was missing dependencies and I was low on time.

Even in your scenario, I'm not sure I'd want a visible message about a security update potentially being displayed to end users. If it was a console.log it could be missed anyway and an alert might scare an end user. 
Perhaps an email list might be more appropriate? Mind you I will be watching the repo, so hopefully I'd pick up any issues through that channel.
